filament/backend/src/opengl/PlatformCocoaGL.mm

```diff
@@ -35,9 +35,15 @@
 using namespace backend;
 
 struct CocoaGLSwapChain : public Platform::SwapChain {
+    CocoaGLSwapChain( NSView* inView );
+    ~CocoaGLSwapChain() noexcept;
+
     NSView* view;
     NSRect previousBounds;
     NSRect previousWindowFrame;
+    __strong NSMutableArray< id<NSObject> >* observers;
```

Comment:
Please fix formatting: remove extra empty spaces (same above in the constructor)